.github/workflows/subtree.yml

```diff
@@ -29,7 +29,42 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: nxtlvlsoftware/git-subtree-action@1.1
+      - uses: pnpm/action-setup@v2.2.4
+
+      - name: Use Node ${{ matrix.node }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        run: |
+          echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
+
+      - uses: actions/cache@v3
+        name: Setup pnpm cache
+        with:
+          path: ${{ steps.pnpm-cache.outputs.pnpm_cache_dir }}
+          key: ${{ runner.os }}-pnpm-store-trpc-main-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-trpc-main-
+
+      - name: Set `shared-workspace-lockfile` to false
+        run: |
+          echo "shared-workspace-lockfile=false" >> .npmrc
+
+      - name: Install dependencies
+        run: |
+          pnpm install
+
+      - name: Commit changes
+        run: |
+          git config --global user.email "email-here"
+          git config --global user.name "trpc-bot"
```

Comment:
not sure what user config you have here @KATT

Reply:
Does the config here matter?

Reply:
I guess not, maybe if you want it to look like a proper commit 🤔

![image](https://user-images.githubusercontent.com/51714798/216734207-041145a2-1c56-4b23-ae90-a663cca3379b.jpeg)